spec/lib/vet360/models/base_spec.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe Vet360::Models::Base do
+  describe 'validation' do
+    let(:email) { build(:email) }
+
+    it 'ensures effectiveEndDate is in the past', aggregate_failures: true do
```

Comment:
Please change this, and everywhere else in this PR, to:

```
..., :aggregate_failures do
```

Per the precedence set in the code base.

Reply:
✅

Reply:
Looks like this last `, aggregate_failures: true` snuck through